dashboard/app/views/devise/registrations/_finish_sign_up.html.haml

```diff
@@ -107,8 +106,7 @@
           - if @user.errors[:us_state].present?
             %p.error= @user.errors[:us_state].join('\n')
         .span5
-          = f.select :us_state, us_state_options, selected: state_code,
-          include_blank: true
+          = f.select :us_state, us_state_options, include_blank: true
```

Comment:
Will this now default to a blank selection?

Reply:
Yes. And if the user tries to create their account with a blank selection, it will report an error telling them to select a State.